README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+# My Introduction
+
+![Picture of me from the summer](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/68551018_2406605662727345_5013656229612879872_o.jpg?_nc_cat=101&_nc_sid=8024bb&_nc_ohc=h5MsPZwZ3-8AX_vzpJu&_nc_ht=scontent-ort2-1.xx&oh=ccf4c5bde72644f8030412a87d017d4a&oe=5EBBAFB4)
+
+Hello! My name is David Mendieta, and I am from Los Angeles, California. I am currently a third-year student majoring in Public Policy, and I aim to attend law school in the near future. While I am sad that I am away from family and friends at the moment, I am very excited to take Computing for the Social Sciences and see how I can better improve my skills in coding and R!
```

Comment:
Because this text is sandwiched between two photos it's pretty easy to gloss over, so make sure you're annotating using **bold**, _italics_, or any of the other Markdown syntaxes